Code/GraphMol/CIPLabeler/configs/Tetrahedral.h

```diff
@@ -0,0 +1,39 @@
+//
+//
+//  Copyright (C) 2020 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#pragma once
+
+#include "Configuration.h"
+
+namespace RDKit {
+namespace CIPLabeler {
+
+class Tetrahedral : public Configuration {
+public:
+  static const int LEFT = 0x1;
+  static const int RIGHT = 0x2;
```

Comment:
I'd like to see a comment describing what these mean

Reply:
I fact, I think I will nix those, and use RDKit's CW/CCW chiral types, and do something similar with the TOGETHER/OPPOSITE constants in the sp2 bonds.